test/fixtures/linter/autofix/bigquery/002_templating/after.sql

```diff
@@ -1,14 +1,17 @@
--- A subset of the hairy test.
-SELECT
-    COUNT(1) AS campaign_count,
-    {{corr_states}}
-    {% for action in considered_actions %}
-        , aaa(
-            bbb(ccc({{metric}}_r, {{action}}), ddd({{metric}}_r)),

```

Comment:
This test specifies `max_line_length` = 50, but this line was 67 characters long. The new result looks correct to me.

Reply:
A reverse case of what you are trying to fix here!

Post templating this is 47-48 characters long as the template values where shorter than the template place holders.

So I agree it's correct that it now splits it with your god change as it's now looking at the placeholder lengths.

Reply:
Exactly. The previous behavior was incorrect, and often very confusing from a user perspective.